code-challenges.js

```diff
@@ -75,33 +84,42 @@
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
+// Create a const "reverse" = .split "currentCohort"
 
-const currentCohort = "Alpha 2023"
+//const currentCohort = "Alpha 2023"
 // Expected output: "3202 ahplA"
 
 // const reverse = currentCohort.split
 // console.log(reverse)
 
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
-// Pseudo code:
+// Pseudo code: Create 2 const given vaules 1,2 labeled as 42,10
+// console log .lastIndexOf with the given values which should produce the expected output  which is their position in the array.
 
-// const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
-// const givenValue1 = 42
-// // Expected output: 7
+//  const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
+//  const givenValue1 = 42
+// // // Expected output: 7
 
-// const givenValue2 = 10
-// // Expected output: 8
-// console.log(indexOf(42, 10))
+//  const givenValue2 = 10
+// // // Expected output: 8
+//  console.log(myNumbers.lastIndexOf(givenValue1))
+//  console.log(myNumbers.lastIndexOf(givenValue2))
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
-// Pseudo code:
+// Pseudo code: Create new const variables for each array
+// Use .sort((a, b) => a-b) to sort and then use .reverse to finalize LARGEST to SMALLEST
```

Comment:
Sort is an excellent approach to this problem!  I would really like to see in your pseudocode more explanation on what is being passed into the sort method.